src/main/kotlin/com/sourcegraph/cody/initialization/EndOfTrialNotificationScheduler.kt

```diff
@@ -48,12 +51,14 @@ class EndOfTrialNotificationScheduler private constructor(val project: Project)
             val codyProTrialEnded =
                 agent.server
                     .evaluateFeatureFlag(GetFeatureFlag.CodyProTrialEnded)
-                    .get(4, TimeUnit.SECONDS) == true
+                    .orTimeout(4, TimeUnit.SECONDS)
+                    .getNow(false) == true
```

Comment:
I've seen the error coming from this place. I managed to get it twice but it's a race (hard to reproduce). This approach will not throw anything.